types/react-color/index.d.ts

```diff
@@ -34,7 +34,7 @@ export interface ColorResult {
     rgb: RGBColor;
 }
 
-export type ColorChangeHandler = (color: ColorResult) => void;
+export type ColorChangeHandler = (color: ColorResult, $event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
This looks OK, but the authors of this DT seem to stick to dfiferent naming convention:
```suggestion
export type ColorChangeHandler = (color: ColorResult, event: React.ChangeEvent<HTMLInputElement>) => void;
```